src/sql/structure.sql

```diff
@@ -1068,9 +1068,14 @@ ALTER TABLE `users2teams`
 ALTER TABLE `users2teams`
   ADD CONSTRAINT `fk_users2teams_teams_id` FOREIGN KEY (`teams_id`) REFERENCES `teams` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
   ADD CONSTRAINT `fk_users2teams_users_id` FOREIGN KEY (`users_id`) REFERENCES `users` (`userid`) ON DELETE CASCADE ON UPDATE CASCADE;
-COMMIT;
 
 
+-- schema 53
```

Comment:
instead of adding it like this, modify the create table statements.

Reply:
Should be solved in the latest changes